.gitlab-ci.yml

```
@@ -185,7 +185,7 @@ default:
           tee ./artifacts/substrate/VERSION;
     fi
   - sha256sum ./artifacts/substrate/substrate | tee ./artifacts/substrate/substrate.sha256
-  - cp -r .maintain/docker/substrate.Dockerfile ./artifacts/substrate/
+  - cp -r scripts/ci/docker/substrate.Dockerfile ./artifacts/substrate/
```

```suggestion
  - cp -r ./scripts/ci/docker/substrate.Dockerfile ./artifacts/substrate/
```
let's be consistent with paths